test/workers/test_base.py

```diff
@@ -95,3 +95,30 @@ def test_enable_registration_with_ctx(hook):
     with hook.local_worker.registration_enabled():
         hook.local_worker.is_client_worker == False
     assert hook.local_worker.is_client_worker == True
+
+
+def test_send_command_whitelist(hook):
```

Comment:
Q: @LaRiffle should I also copy those two tests in the ```test_websocket_worker```

Reply:
would be a good idea!
just an info:
prefer to do:
```
def test_send_command_whitelist(workers):
    bob  = workers["bob"]
    ...
```